ee/session_recordings/session_recording_extensions.py

```diff
@@ -216,3 +215,12 @@ def persist_recording_v2(recording_id: str, team_id: int) -> None:
         recording.full_recording_v2_path = target_key
         recording.save()
         SNAPSHOT_PERSIST_SUCCESS_V2_COUNTER.inc()
+
+
+def persist_recording_v2(recording_id: str, team_id: int) -> None:
+    """Persist a recording to S3 using the v2 format"""
+    try:
+        _persist_recording_v2_impl(recording_id, team_id)
```

Comment:
i think clearer just wrapped in a try catch than having `*_impl` method but 🤷

Reply:
I went with this because I wanted to keep metric calls in one place and didn't want to have a one big try catch block.